instance.cc

```diff
@@ -2994,13 +3053,39 @@ Heightfield PGInstance::getHeightField(float bx, float bz) {
 
         float elevation = elevationSum / totalHeightFactors;
         localHeightfield.heightField = elevation;
+
+        // materials
+
+        getComputedMaterials(localHeightfield, materialsCounts, totalMaterialFactors, fWorldPosition);
+
+        std::vector<uint8_t> seenMaterials;
+        for (auto kv : materialsCounts)
+        {
+            unsigned char b = kv.first;
+            seenMaterials.push_back(b);
+        }
```

Comment:
II suspect the intent was to sort these, but they are not sorted for some reason.